io/shared/src/main/scala/fs2/io/file/Files.scala

```diff
@@ -409,6 +415,34 @@ sealed trait Files[F[_]] extends FilesPlatform[F] {
       limit: Long,
       flags: Flags
   ): Pipe[F, Byte, Nothing]
+
+  /** Writes to the specified file as an utf8 string.
+    *
+    * The file is created if it does not exist and is truncated.
+    * Use `writeUtf8(path, Flags.Append)` to append to the end of
+    * the file, or pass other flags to further customize behavior.
+    */
+  def writeUtf8(path: Path): Pipe[F, String, Nothing] = writeUtf8(path, Flags.Write)
+
+  /** Writes to the specified file as an utf8 string using
+    * the specified flags to open the file.
+    */
+  def writeUtf8(path: Path, flags: Flags): Pipe[F, String, Nothing] = in =>
+    in.through(text.utf8.encode).through(writeAll(path, flags))
```

Comment:
Should we add this? I'm not entirely sure how `Write` and `Append` flags interact 🤔 

```suggestion
    in.through(text.utf8.encode).through(writeAll(path, flags.addIfAbsent(Flag.Write)))
```

Reply:
Wait, nevermind, we don't need this. It's already added further down the call-chain :)